### DISCUSSION TITLE
CPANEL

### DISCUSSION THREAD

**Discussion by AdrianooTavares** [RamonSilva20/mapos#1735] [General]
Esta dando esse erro não estou conseguindo instalar!

Sua instalação está incompleta, por favor instale as dependências utilizando o composer. A pasta system não foi localizada.

**Comment by Pr3d4dor** [comment]
@AdrianooTavares Você precisa instalar as dependencias do projeto usando composer

**Comment by mjrbrasil** [comment]
Consigo instalar o MAPOS direto na hospedagem sem ser local?

**Reply by almron** [reply]
Consegue sim. Eu utilizo o HostGator de forma bem fácil.

**Comment by mjrbrasil** [comment]
Maravilha! Poderia explicar como instalar? Tbm utilizo hostgator

Obter o Outlook para iOS<https://aka.ms/o0ukef>
________________________________
De: almron ***@***.***>
Enviado: Friday, September 23, 2022 10:23:04 PM
Para: RamonSilva20/mapos ***@***.***>
Cc: mjrbrasil ***@***.***>; Comment ***@***.***>
Assunto: Re: [RamonSilva20/mapos] CPANEL (Discussion #1735)


Consegue sim. Eu utilizo o HostGator de forma bem fácil.

**Reply by almron** [reply]
Entre em Portal do Cliente / Produtos / CPanel.
Crie um Domínio e atribua um Certificado SSL/TLS
--> Exemplo: meusite.com.br
Crie uma Pasta ou Subdomínio
--> Exemplo https://meusite.com.br/mapos ou https://mapos.meusite.com.br
No CPanel, vá em Gerenciador de arquivos. Na pasta criada, transfira o arquivo compactado do Mapos no [Github](https://github.com/RamonSilva20/mapos/archive/refs/heads/master.zip) e extraia os arquivos.
No CPanel, vá em Banco de Dados MySQL, crie um usuário e um banco de dados. Anote o usuário e senha e o nome e endereço do bando de dados criado.
Através do Suporte Técnico da HostGator, peça para liberar o acesso ssh para o seu domínio. Isso será necessário para executar o a instalação do Composer.

Siga a instrução abaixo:
https://github.com/RamonSilva20/mapos/blob/master/README.md

**Comment by mjrbrasil** [comment]
Obrigado!!!

Obter o Outlook para iOS<https://aka.ms/o0ukef>
________________________________
De: almron ***@***.***>
Enviado: Sunday, September 25, 2022 4:35:58 PM
Para: RamonSilva20/mapos ***@***.***>
Cc: mjrbrasil ***@***.***>; Comment ***@***.***>
Assunto: Re: [RamonSilva20/mapos] CPANEL (Discussion #1735)


Entre em Portal do Cliente / Produtos / CPanel.
Crie um Domínio e atribua um Certificado SSL/TLS
--> Exemplo: meusite.com.br
Crie uma Pasta ou Subdomínio
--> Exemplo https://meusite.com.br/mapos ou https://mapos.meusite.com.br
No CPanel, vá em Gerenciador de arquivos. Na pasta criada, transfira o arquivo compactado do Mapos no Github<https://github.com/RamonSilva20/mapos/archive/refs/heads/master.zip> e extraia os arquivos.
No CPanel, vá em Banco de Dados MySQL, crie um usuário e um banco de dados. Anote o usuário e senha e o nome e endereço do bando de dados criado.
Através do Suporte Técnico da HostGator, peça para liberar o acesso ssh para o seu domínio. Isso será necessário para executar o a instalação do Composer.

Siga a instrução abaixo:
https://github.com/RamonSilva20/mapos/blob/master/README.md
